release-cloudbuild-preparation.yaml

```diff
@@ -77,7 +77,6 @@ steps:
           --platform linux/amd64 \
           --build-arg BASE_IMAGE=$$USERNAME/prepare-release:base-${_ZENML_NEW_VERSION} \
           --build-arg CLOUD_PROVIDER=gcp \
-          --build-arg ZENML_BRANCH=${_ZENML_BRANCH} \
```

Comment:
I am confused about all this. Maybe you can clear it for me. I see now that we don't even build any quickstart images for the release (which I like). But if that's the case, why do we even need to build them now for testing the release? Isn't ok for us to use the base image to test?

Previously, different quickstarts required a different "pre-specified" list of requirements to run the quickstart. As far as I can see, this is no longer a concern of the new one.